src/util/services/proton.service.ts

```diff
@@ -24,13 +24,21 @@ class ProtonSDK {
 
   login = async () => {
     try {
-      this.link = await ConnectWallet({ chainId: this.chainId, endpoints: this.endpoints }, { requestAccount: this.requestAccount }, this.appName, ProtonVIPLogo);
+      this.link = await ConnectWallet({
```

Comment:
This is still using  the  old proton-web-sdk package. You can update it to the new version (v2.5.2) with an npm update. The arguments  here are different.